### DISCUSSION TITLE
Ignore Drafts

### DISCUSSION THREAD

**Discussion by hrzafer** [pngwn/MDsveX#446] [Q&A]
Let's say I have some `.md` files that are not ready to be published but I want them to remain as drafts. Is there a way to tell mdsvex to ignore such files.

**Comment by pngwn** [comment]
Telling mdsvex about this would really help much because kit see them regardless and would generate a route for them (all mdsvex could do is perhaps remove the content).

Currently the best solution for this is to use the [`routes` configuration option in sveltekit](https://kit.svelte.dev/docs/configuration#routes): read the files, check their metadata and return false if `draft: true` in the frontmatter. 

Alternatively, you could set the name to something like `post.draft.svx` and you can use the filename to decide whether to ignore the route or not:

```js
export default {
 ...kit_config,
 routes: (filepath) => !/.*draft.svx/.test(filepath)
}
```

**Comment by spences10** [comment]
I use a frontmatter property, `isPrivate: true` then don't render out files that that are marked as private/draft/not complete

This is just client side and your unpublished posts will still be visible to search bots 😊

I prefer this approach so I can share unpublished posts/drafts via a URL but they're easy to find on the site.

**Comment by rchrdnsh** [comment]
I just add a status field to the frontmatter. I use 'draft', 'published', 'featured', and 'marquee' for showing the content in different and/or more than one place.
